src/main/kotlin/no/nav/familie/ef/personhendelse/inntekt/vedtak/EfVedtakRepository.kt

```diff
@@ -104,4 +106,5 @@ data class Inntektsendring(
     val harNyttVedtak: Boolean,
     val harEndretInntekt: Boolean,
     val prosessertTid: LocalDateTime,
+    val inntektEndretProsent: Int,
```

Comment:
Skal det hentes opp tidligere data for denne? For data som allerede er i databasen så er denne null?

Reply:
Defaulter til 0 i migreringscriptet